core/src/kura.rs

```diff
@@ -86,6 +87,14 @@ impl Kura {
         Ok(kura)
     }
 
+    #[allow(clippy::unwrap)]
+    pub fn blank_kura_for_testing() -> (Arc<Kura>, ThreadHandler, tempfile::TempDir) {
+        let dir = tempfile::tempdir().unwrap();
+        let kura = Self::new(Mode::Strict, dir.path(), 100, false).unwrap();
+        let th = Self::start(kura.clone());
```

Comment:
While I can see it from the type, you should try and make the naming of things as clear as possible. `thread_handle` is sufficient.